magento/rest.py

```diff
@@ -13,10 +14,24 @@ def __init__(self, url, token, verify_ssl=True):
         self._token = token
         self._verify_ssl = verify_ssl
 
-    def call(self, resource_path, arguments):
+    def call(self, resource_path, arguments, http_method=None, storeview=None):
         url = '%s/%s' % (self._url, resource_path)
-        res = requests.get(
-            url, params=arguments, verify=self._verify_ssl,
-            headers={'Authorization': 'Bearer %s' % self._token})
+        if storeview:
+            # https://github.com/magento/magento2/issues/3864
+            url = url.replace('/rest/V1/', '/rest/%s/V1/' % storeview)
+        if http_method is None:
+            http_method = 'get'
```

Comment:
Why not set `get` as the default value of `http_method` parameter and remove these two lines?

Reply:
Makes sense ;-) Will test that refactoring this week.

Reply:
@sebalix I gave it a try but I didn't like it. The super method in api.py passes http_method explicitely, so I'd have to set the default in that method's signature where it does not make a lot of sense.